src/main/java/org/example/service/HelloSender.java

```diff
@@ -0,0 +1,15 @@
+package org.example.service;
+
+import java.io.PrintWriter;
+
+public class HelloSender {
+    public void helloSender(PrintWriter printWriter) {
```

Comment:
Firstly, create the result string and call the `printWriter.write` only once at the end.